apps/game_backend/lib/game_backend/curse_of_mirra/quests.ex

```diff
@@ -65,6 +70,43 @@ defmodule GameBackend.CurseOfMirra.Quests do
     Quest.changeset(quest, attrs)
   end
 
+  def get_daily_quest(daily_quest_id) do
+    q = from(dq in DailyQuest, preload: [:quest], where: dq.id == ^daily_quest_id)
+
+    Repo.one(q)
+  end
+
+  def get_users_daily_quests(user_id) do
+    q = from(dq in DailyQuest, preload: [:quest], where: dq.user_id == ^user_id)
+
+    Repo.all(q)
+  end
```

Comment:
Add @doc tags for these functions please.

Reply:
Done!